sql/plan/bindvar.go

```diff
@@ -28,7 +28,7 @@ import (
 // This applies binding substitutions across *SubqueryAlias nodes, but will
 // fail to apply bindings across other |sql.Opaque| nodes.
 func ApplyBindings(ctx *sql.Context, n sql.Node, bindings map[string]sql.Expression) (sql.Node, error) {
-	withSubqueries, err := TransformUp(n, func(n sql.Node) (sql.Node, error) {
+	return TransformUp(n, func(n sql.Node) (sql.Node, error) {
```

Comment:
i think a BindVar application was being missed, if this needs to be more discriminate I can try to refactor

Reply:
This needs to hit other Opaque nodes too, write a test.